WebApp/handler/SystemAdministrationHandler.py

```diff
@@ -248,7 +253,7 @@ def web_hostAction( self ):
         result = yield self.threadTask( client.restartComponent, str( "*" ) , str( "*" ) )
       elif action == "revert":
         result = yield self.threadTask( client.revertSoftware )
-      elif action == "updat":
+      elif action == "update":
```

Comment:
You have to change the "action" in the javascript as well.

Reply:
Done.